torch/csrc/jit/api/module_save.cpp

```diff
@@ -5,35 +5,45 @@ namespace torch {
 namespace jit {
 
 void Module::save(std::ostream& out, const ExtraFilesMap& extra_files) const {
-  ExportModule(*this, out, extra_files, false /* bytecode_format */);
+  ExportModule(
+      *this,
+      out,
+      extra_files,
+      at::optional<uint64_t>() /* bytecode_format version*/);
 }
 
 void Module::save(const std::string& filename, const ExtraFilesMap& extra_files)
     const {
-  ExportModule(*this, filename, extra_files, false /* bytecode_format */);
+  ExportModule(
+      *this,
+      filename,
+      extra_files,
+      at::optional<uint64_t>() /* bytecode_format version*/);
 }
 
 void Module::_save_for_mobile(
     std::ostream& out,
     const ExtraFilesMap& extra_files,
+    at::optional<uint64_t> version,
     bool save_mobile_debug_info) const {
   ExportModule(
       *this,
       out,
       extra_files,
-      true /* bytecode_format */,
```

Comment:
what did this bool do?

Reply:
My understanding is, it's used to control whether bytecode will be exported or not. When exporting jit model (`torch.jit.save`), it's set to `false`. When exporting bytecode model, (`torch.jit_save_for_mobile`), it's set to `true`. @iseeyuan can confirm?

Reply:
Right. Most of the serialization codes are shared between full jit and lite, except for the bytecode part. This bool controls if bytecode is saved or not. Basically, 
jit.save() {
  ExportModule(..., false);
}

_save_for_mobile() {
  ExportModule(..., false);
}